build/yaml/pipelines/codeanalysis.yaml

```diff
@@ -15,7 +15,7 @@ schedules:
   displayName: Weekly Static Analysis
   branches:
     include:
-    - main
+    - release
```

Comment:
We can probably get rid of this. We run static analysis as a CI after each completed pull request now.

Reply:
Removed